lib/banken.rb

```diff
@@ -56,15 +56,15 @@ def banken_user
   end
 
   def skip_authorization
-    @_banken_loyalty_authorized = true
+    @_banken_authorization_performed = true
   end
 
   def verify_authorized
-    raise AuthorizationNotPerformedError unless banken_loyalty_authorized?
+    raise AuthorizationNotPerformedError unless banken_authorization_performed?
   end
 
-  def banken_loyalty_authorized?
-    !!@_banken_loyalty_authorized
+  def banken_authorization_performed?
```

Comment:
Since the public method `banken_loyalty_authorized?` is renamed, The old method should generate the warning like the following code instead of removing it so that nobody face with `NoMethodError`.

```ruby
def banken_loyalty_authorized?
  ActiveSupport::Deprecation.warn('banken_loyalty_authorized? is deprecated and will be removed soon, Use banken_authorization_performed? instead.')
  banken_authorization_performed?
end
```

Reply:
thanks and don't forget to update README.md :wink: https://github.com/kyuden/banken/blame/master/README.md#L205